docs/advanced/picking.md

```diff
@@ -73,12 +73,36 @@ calculatePickingColors(attribute) {
 
 ## Custom Shaders
```

Comment:
Maybe call this `Implementing Picking in Custom Shaders`. 

Longer term, I am thinking that maybe we should collect such information in a separate section on shader writing.